content/pt-br/kubernetes.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Kubernetes
+status: Completed
+category: tecnologia
+tags: ["infrastructure", "fundamental", ""]
+---
+
+## O que é
+
+Kubernetes, normalmente abreviado como K8s, é um orquestrador de containers de código aberto. 
+Ele automatiza o ciclo de vida de aplicações em containers em infraestruturas modernas, funciona como um "sistema operacional de datacenter" que gerencia aplicativos em um [sistema distribuído](/distributed-systems/).
+
+O Kubernetes planeja os [contêineres](/container/) entre [nós](/nodes/) de um [cluster](/cluster/), agrupando vários recursos de infraestrutura como balanceador de carga, armazenamento persistente, etc., para executar aplicações em containers.
```

Comment:
Redirects to the Portuguese pages in the /container/ /node/ and /cluster/ links by adding /pt-br/